css/amp.css

```diff
@@ -365,6 +370,12 @@ amp-analytics {
   visibility: hidden;
 }
 
+/**
+ * Allow images inside amp-img to be fit-cover: object.
+ */
+amp-img.-amp-object-fit-cover > img {
```

Comment:
Ah. Inside YouTube - I guess no longer since we reverted back to the hack instead of using `amp-img` I can drop this.